frontend/src/Button.css

```diff
@@ -0,0 +1,7 @@
+.Button {
```

Comment:
Yeah, with nested components I haven't found it to be an issue. The pattern continues and the new child component just gets its own scss file. One thing that I started doing (I think more in the PR #25) is writing mixins in /css/*.scss files and including them in component scss files. To avoid repeating ourselves too much.